src/channel.ts

```diff
@@ -1097,16 +1097,17 @@ export class Channel<
   }
 
   /**
-   * countUnread - Count the number of messages with a date thats newer than the last read timestamp
+   * countUnread - Count of unread messages
    *
    * @param {Date | Immutable.ImmutableDate | null} [lastRead] lastRead the time that the user read a message, defaults to current user's read state
    *
    * @return {number} Unread count
    */
   countUnread(lastRead?: Date | Immutable.ImmutableDate | null) {
```

Comment:
`countUnread` function fails to return actual count of unread messages if [number of unread messages] > [number of messages loaded/queried in channel]. E.g., if user queries only first page of channel messages and user has 100 unread messages --->  `channel.countUnread()` will return only 25.